backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/ddl/functions/Docs_DeliveryInstructions_ProductDetails.sql

```diff
@@ -22,7 +22,7 @@ SELECT wh.name                                  AS warehouseName,
        p.grade,
        COALESCE(uomt.uomsymbol, uomt.uomsymbol) AS uom
 FROM M_ShipperTransportation st
-         JOIN m_delivery_planning dp ON dp.releaseno = st.documentno
+         JOIN m_delivery_planning dp ON dp.m_delivery_planning_id = st.m_delivery_planning_id
```

Comment:
in case you didn't yet - please make sure to have an index on `M_ShipperTransportation.m_delivery_planning_id`